app/admin/categories.rb

```diff
@@ -8,26 +8,62 @@
   index do
     id_column
     column :name
+    column :short_name
     column :description
+    column "Parent Category" do |p|
+      if !p.nil?
+        Category.find_by_id(p.parent_category_id)
+       end
+    end
     column :related_terms
+    column :is_other
     actions
   end
 
   show do
     attributes_table do
       row :id
       row :name
+      row :short_name
       row :description
+      row "Parent Category" do |p|
+        if !p.nil?
+          parent_cat = Category.find_by_id(p.parent_category_id)
+        end
+      end
       row :related_terms
+      row :is_other
+
+      category_practice = CategoryPractice.where(category_id: params["id"])
+      practices = []
+      cat = Category.find_by_id(params["id"]).name
+      category_practice.each do |cat_pract|
+        practices << Practice.find_by_id(cat_pract.practice_id).name
+      end
+      practices_with_cat = ""
+      practices.each do |practice|
+        practices_with_cat += practice + ", "
+      end
+      if practices_with_cat.length > 0
```

Comment:
do we want to display this?

Reply:
Good catch... Elisa - ur right the team decided not to display that info.  I removed it.